main.py

```diff
@@ -37,7 +36,7 @@ def reEmail(fname):
 def reAward(fname):
   fhand = open(fname, 'r')
   for line in fhand:
-    extr = re.findall('.*', line)
+    extr = re.findall('-\s.*', line)
```

Comment:
Your regex will extract extra in front of the award winner.  "-  Usain Bolt". Use brackets.
extr = re.findall('- (.*)', line)